### DISCUSSION TITLE
使用pixiv.re在线反代时无法发送图片

### DISCUSSION THREAD

**Discussion by HRxiaohu** [Tsuk1ko/CQ-picfinder-robot#367] [问题求助] [answered]
使用pixiv.re进行在线反代时提示消息为空无法发送图片，访问图片直链时网页白屏。
无法确定是机器人问题还是网站问题。

**Comment by HRxiaohu** [comment]
本地反代时也提示这个消息

**Comment by Tsuk1ko** [comment, marked as answer]
可能是你机器网络问题，正常情况下直接访问 pixiv.re 应当可以看到网页

![image](https://user-images.githubusercontent.com/24877906/196664391-aba92d34-a123-40d2-83e2-11cc8bf7385e.png)

本地反代也对网络环境有一定要求，毕竟相当于直连P站

可以尝试使用本地反代同时配置代理 `bot.proxy`

**Reply by HRxiaohu** [reply]
![image](https://user-images.githubusercontent.com/63327671/196665174-d24ecd4c-5d22-4a62-83b5-b65ec6520352.png)

**Reply by HRxiaohu** [reply]
主站没有问题，可是到图片直链时会白屏
https://i.pixiv.re/img-original/img/2019/01/16/01/49/12/72685648_p0.jpg

**Reply by Tsuk1ko** [reply]
看来是 i.pixiv.re 出问题了，先用本地反代加代理吧

**Reply by Tsuk1ko** [reply]
用 i.loli.best 也行，这个是我的在线反代服务，不过不保证一直能用就是了

**Reply by HRxiaohu** [reply]
好的，感谢大佬。
